pages/docs.json

```diff
@@ -62,6 +62,13 @@
           "title": "TypeScript"
         }
       ]
+    }, {
+      "title": "FAQ's and Tips & Tricks",
```

Comment:
Can you rename it to be shorter maybe :/ We don't really need to call it "Tips & Tricks". We can just reword all the subsections to be questions.

e.g. "How do I nest selectors?"

![localhost-3000-docs 2](https://cloud.githubusercontent.com/assets/2041385/26726068/7b0cb076-4799-11e7-89fa-2c9c17847cb6.png)